tests/streaming_test.py

```diff
@@ -535,6 +559,18 @@ async def test_account_activity_subs_failure(self, ws_connect):
         with self.assertRaises(tda.streaming.UnexpectedResponseCode):
             await self.client.account_activity_sub()
 
+    @no_duplicates
```

Comment:
This one might also be unrealistic. If you can figure out a way to trigger this, great.